casadm/cas_lib.c

```diff
@@ -1829,9 +1829,11 @@ int check_if_mounted(int cache_id, int core_id)
 	}
 
 	while ((mstruct = getmntent(mtab)) != NULL) {
+		mnt_fsname_len = strnlen(mstruct->mnt_fsname,
+				(sizeof("/dev/") + sizeof(mstruct->mnt_fsname)));
```

Comment:
This would always return the same value which is not buffer length. See my comment with explanation above.

Reply:
Do You think that `(sizeof("/dev/") + sizeof(mstruct->mnt_fsname))` could be replaced here with `sizeof(dev_buf)`? CAS wouldn't compare longer strings anyway.

Reply:
`sizeof(dev_buf)` is good enough. CAS device names should never be longer than this.